docs/docs/reference/changed-features/pattern-matching.md

```diff
@@ -153,26 +152,6 @@ object Nat {
 // 5 is a natural number
 ```
 
-## Name-based Match
```

Comment:
Looks like the removal of this section allows to remove the last paragraph of this file ("There are plans for further simplification ...")